src/bin/pg_dump/dump_babel_utils.c

```diff
@@ -53,9 +53,9 @@ typedef enum {
 static babelfish_status bbf_status = NONE;
 
 static char *default_bbf_db_principals =
-			"('master_dbo', 'master_db_owner', 'master_guest', 'master_db_accessadmin', "
-			"'msdb_dbo', 'msdb_db_owner', 'msdb_guest', 'msdb_db_accessadmin', "
-			"'tempdb_dbo', 'tempdb_db_owner', 'tempdb_guest', 'tempdb_db_accessadmin') ";
+			"('master_dbo', 'master_db_owner', 'master_guest', 'master_db_accessadmin', 'master_db_securityadmin', "
+			"'msdb_dbo', 'msdb_db_owner', 'msdb_guest', 'msdb_db_accessadmin', 'msdb_db_securityadmin', "
+			"'tempdb_dbo', 'tempdb_db_owner', 'tempdb_guest', 'tempdb_db_accessadmin', 'tempdb_db_securityadmin') ";
```

Comment:
Instead of updating this static list everytime, is it possible to get this list dynamically using some query during dump? same is applicable for default_bbf_roles

Reply:
It should be possible IMO, We can rely on the is_fixed_role metadata. Currently, we are not utilizing it and not even updating it for fixed roles.

Reply:
Lets open internal jira to track this work.